hypha/static_src/javascript/determination-template.js

```diff
@@ -69,7 +69,7 @@
     });
     window.addEventListener("load", function (event) {
         const proposal_form_field = document.querySelector("#id_proposal_form");
```

Comment:
It's probably not important, but how or where does `id_proposal_form` come from? I could not find it anywhere.

Reply:
See https://github.com/HyphaApp/hypha/blob/a5d3a4f5e8257af713683cfcbe3e37719d4fa152/hypha/apply/determinations/forms.py#L368-L393